dev/PushNotifications/PushNotificationReceivedEventArgs.cpp

```diff
@@ -90,7 +87,7 @@ namespace winrt::Microsoft::Windows::PushNotifications::implementation
 
     bool PushNotificationReceivedEventArgs::Handled()
     {
-        if (m_isBIAvailable)
+        if (m_isUnpackagedApp)
```

Comment:
Wait, are the code blocks in their opposite side? I would expect to return `m_handledUnpackaged` if this is an unpkg app.

Reply:
I would just rename this variable to `m_isPackagedApp`. The code works as if that were it's real name.